.github/workflows/issues.yml

```
@@ -0,0 +1,20 @@
+# A workflow to add bug issues to the roadmap project
+# https://github.com/actions/add-to-project
+
+name: Add bugs to roadmap project
```

```suggestion
name: Issues
```

Since this will show up in the list of workflows, let's keep it short